config/admin_ranks.txt

```diff
@@ -43,15 +45,13 @@ Game Master		= +EVERYTHING *EVERYTHING
 Lazy Master 	= +EVERYTHING -AUTOLOGIN *EVERYTHING
 Three Dog		= -@ -ADMIN +DJ
 
-Head Coder      = +FUN +STEALTH +POSSESS +REJUV +BUILD +SERVER +DEBUG +VAREDIT +SOUND +SPAWN +RIGHTS +DJ +POLL
-Coder			= +DEBUG +VAREDIT +SERVER +SPAWN +POLL -AUTOLOGIN
+Scribe 			= +SERVER +DEBUG +VAREDIT +SPAWN -AUTOLOGIN
+Head Scribe 	= +@ +POLL -AUTOLOGIN
 
-Initiate		= +ADMIN +BAN +REJUV 
-Aspirant 		= +ADMIN +BAN +REJUV +SPAWN +SOUND
-Knight 			= +ADMIN +BAN +REJUV +STEALTH +FUN +POSSESS +SOUND +SPAWN +VAREDIT
-Paladin			= +ADMIN +BAN +REJUV +STEALTH +FUN +POSSESS +BUILD +SOUND +SPAWN +SERVER +DJ +POLL +VAREDIT
-Head Paladin	= +ADMIN +BAN +REJUV +STEALTH +FUN +POSSESS +BUILD +SOUND +SPAWN +SERVER +RIGHTS +DJ +POLL +VAREDIT
-Scribe 			= +STEALTH +POSSESS +REJUV +BUILD +DEBUG +VAREDIT +SOUND +SPAWN +DJ +POLL -AUTOLOGIN
-Head Scribe 	= +STEALTH +POSSESS +REJUV +BUILD +SERVER +DEBUG +VAREDIT +SOUND +SPAWN +RIGHTS +DJ +POLL -AUTOLOGIN
-Elder			= +EVERYTHING
-Host 			= +EVERYTHING *EVERYTHING
+Initiate		= +ADMIN
+Aspirant 		= +@ +BAN
```

Comment:
Same issues here due to host and the like being higher up in the text file

Reply:
We should probably be explicit about what everyone has though, I don't like +@ as it creates this kind of confusion.

Reply:
+@ is done to reduce bloat if folk cant wrap there brain around what the line above has in terms of perms they should not be messing around with it.

As for your comment Nappist. this is how it is in the live server, right. Now.